WooCommerce/Classes/Analytics/TracksProvider.swift

```diff
@@ -66,27 +66,27 @@ private extension TracksProvider {
         let currentAnalyticsUsername = UserDefaults.standard[.analyticsUsername] as? String ?? ""
         let anonymousID = ServiceLocator.stores.sessionManager.anonymousUserID
         if ServiceLocator.stores.isAuthenticated,
-            let account = ServiceLocator.stores.sessionManager.defaultAccount,
-            let credentials = ServiceLocator.stores.sessionManager.defaultCredentials {
+           let account = ServiceLocator.stores.sessionManager.defaultAccount,
+           case let .wpcom(username: _, authToken: authToken, siteAddress: _) = ServiceLocator.stores.sessionManager.defaultCredentials {
```

Comment:
Nit: this can be simplified:

```suggestion
           case let .wpcom(_, authToken, _) = ServiceLocator.stores.sessionManager.defaultCredentials {
```

Reply:
Done as part of https://github.com/woocommerce/woocommerce-ios/pull/8446/commits/b231bcef7029ee63fab0c1e723c9e645ed2e16e1